File: PagingRoomJava/app/src/main/java/com/websarva/wings/android/pagingroomjava/data/local/Phone.java
package com.websarva.wings.android.pagingroomjava.data.local;

import androidx.annotation.NonNull;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "phones")
public class Phone {
	@PrimaryKey(autoGenerate = true)
	public long id;
	@NonNull
	public String phoneNo;
}

File: PagingRoomKotlin/app/src/main/java/com/websarva/wings/android/pagingroomkotlin/data/local/Phone.kt
package com.websarva.wings.android.pagingroomkotlin.data.local

import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "phones")
data class Phone(
	@PrimaryKey(autoGenerate = true)
	val id: Long,
	val phoneNo: String
)
